backend/db/seeders/20240522201100-demo-spots.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+const { Spot } = require('../models');
+const bcrypt = require("bcryptjs");
+
+/** @type {import('sequelize-cli').Migration} */
+
+let options = {};
+options.tableName = 'Spots'
+if (process.env.NODE_ENV === 'production') {
+  options.schema = process.env.SCHEMA;  // define your schema in options object
+}
+
+//BONUS REMINDER: IMPLEMENT ZIP CODES
```

Comment:
You do not need to implement zip codes, as this is something you could later manage with the google maps api

Reply:
Thank you for the clarification, yeah I don't need to reinvent the wheel if there are APIs that I can integrate to my project. I will search about Google Maps API for a future integration.